test/concurrent/ThreadPoolTest.ooc

```diff
@@ -65,6 +65,8 @@ ThreadPoolTest: class extends Fixture {
 			comparison := t"fail"
 			result := future wait(comparison)
 			expect(result == comparison)
+			Time sleepMilli(1000)
```

Comment:
We can currently not `wait` on a `Future` we have previously called `cancel` on. This is ugly but works for now. We will get something better with #1588 or similar.